ImplementationGuide/markdown/Uebersicht.md

```diff
@@ -1,11 +1,11 @@
 
 
 **Autorisierung von Zugriffen auf FHIR Ressourcen**
 
-Ziel der vorliegenden Spezifikation ist es, per FHIR RESTful API formulierte Anfragen an einen Ressourcenserver unter Nutzung eines zuverlässigen und sicheren Autorisierungsprotokolls - und damit unter Berücksichtigung von vergebenen Berechtigungen und formulierten Sicherheitsmechanismen - zu beantworten. Hierzu werden Teile des von _SMART Health IT_ und _Boston Childrens Hospital_ auf Basis des OAuth2-Standards definierten, von HL7 als Teil des FHIR-Standards übernommenen [_'SMART on FHIR' API_](https://smarthealthit.org/smart-on-fhir-api/) profiliert, so dass diese im Kontext der bestehenden ISiK-Festlegen nutzbar sind. Das SMART-on-FHIR-API soll die Weiterentwicklung von _Electronic Health Records_ (EHR) zu Plattformen befördern, bei denen - analog zu mobilen Plattformen wie IOS oder Android - Anwendungen aus einem _Store_ in eine solche Plattform eingebracht und dort in einer sicheren Umgebung ausgeführt werden (_"SMART App Launch"_).
+Ziel der vorliegenden Spezifikation ist es, per FHIR RESTful API formulierte Anfragen an einen Ressourcenserver unter Nutzung eines zuverlässigen und sicheren Autorisierungsprotokolls - und damit unter Berücksichtigung von vergebenen Berechtigungen und formulierten Sicherheitsmechanismen - zu beantworten. Hierzu werden Teile des von _SMART Health IT_ und _Boston Childrens Hospital_ auf Basis des OAuth2-Standards definierten, von HL7 als Teil des FHIR-Standards übernommenen [_'SMART on FHIR' API_](https://smarthealthit.org/smart-on-fhir-api/) profiliert, so dass diese im Kontext der bestehenden ISiK-Festlegen nutzbar sind. Das SMART-on-FHIR-API soll die Weiterentwicklung von _Electronic Health Records_ (EHR) zu Plattformen befördern, bei denen - analog zu mobilen Plattformen wie iOS oder Android - Anwendungen aus einem _Store_ in eine solche Plattform eingebracht und dort in einer sicheren Umgebung ausgeführt werden (_"SMART App Launch"_). Das Ziel von ISiK ist es an dieser Stelle die Einbindung dieser Drittsysteme standardisiert zu ermöglichen, sodass nicht für jedes bestätigungsrelevante System eine eigene Form der Autorisierung implementiert werden muss.
```

Comment:
suggestion: folgendes ggf. in einen eigenen Abschnitt "Ziele" integrieren: "Das Ziel von ISiK ist es an dieser Stelle die Einbindung dieser Drittsysteme standardisiert zu ermöglichen, sodass nicht für jedes bestätigungsrelevante System eine eigene Form der Autorisierung implementiert werden muss"

zu klären bleibt aber:
question: Wer hat hier diesen Bedarf (angemeldet)?